PhpAmqpLib/Wire/IO/StreamIO.php

```diff
@@ -256,7 +256,7 @@ public function write($data)
             }
 
             set_error_handler(array($this, 'error_handler'));
-            $buffer = fwrite($this->sock, $data);
+            $buffer = fwrite($this->sock, $data, 8192);
```

Comment:
Thanks for the feedback. OpenSSL's SSL_write() function has been known to misbehave for quite some time now when passed a buffer larger than 8,192 bytes in length (see [this mailing list post](http://comments.gmane.org/gmane.comp.encryption.openssl.user/4361) from September, 2002). I'll add a comment to the source explaining this.